resources/lib/relive.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+from __future__ import print_function, division, absolute_import
+
+from .helpers import maybe_json
+from .kodi import log
+
+
+class Relives(object):
+    def __init__(self, json):
+        self.recordings = [ReliveItem(elem) for elem in json]
+
+
+class ReliveItem(object):
+    def __init__(self, json):
+        self.index_url = json['index_url']
+        # self.media_conference_id = json['media_conference_id']
+        self.project = json['project']
+        self.updated_at = json['updated_at']
+
+
+class ReliveRecording(object):
+    def __init__(self, json):
+        self.duration = json['duration']
+        self.guid = json['guid']
+        self.id = json['id']
+        self.mtime = json['mtime']
+        self.playlist = json['playlist']
+        self.release_url = maybe_json(json, 'release_url', '')
+        self.mp4 = maybe_json(json, 'mp4', '')
+        self.room = json['room']
+        self.start = json['start']
+        self.status = json['status']
+        self.thumbnail = json['thumbnail']
+        self.title = json['title']
```

Comment:
not sure which fields are mandatory or which other fields could occur as I didn't find an API documentation and just looked at https://live.alb.c3voc.de/relive/rc3/index.json

Reply:
I think it might be generated by these scripts: https://github.com/voc/hls-relive#genpagepl - It would probably be best to restrict our "model" to the minimum fields needed or use `maybe_json` liberally 😉

Reply:
yeah, I will strip out all the unused fields from the model